content/en/blog/2023/otel-in-focus-01.md

```diff
@@ -0,0 +1,84 @@
+---
+title: OpenTelemetry in Focus - January 2023
+linkTitle: OTel in Focus - 2023/01
+date: 2023-01-31
+author: >-
+  [Austin Parker](https://github.com/austinlparker)
+---
+
+Welcome to the first edition of OpenTelemetry In Focus! This blog is intended to
+be an overview of important releases, roadmap updates, and community news. This
+series will be focusing on our core components, such as the specification, data
+format, tools, and most popular API/SDKs.
+
+Are you a maintainer with something you’d like featured here? Get in
+touch with me [via email](mailto:austin@lightstep.com), or on the CNCF Slack!
```

Comment:
```suggestion
touch with me [via email](mailto:austin@lightstep.com), or on the
[CNCF Slack, #otel-comms](https://cloud-native.slack.com/archives/C02UN96HZH6) channel.
```

https://cloud-native.slack.com/archives/C02UN96HZH6

Reply:
☝️ @austinlparker - is it ok to refer folks to the `#otel-comm` channel, or were you expecting a DM via Slack?